File: c#/Monopoly/Monopoly/Game.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Monopoly
{        
    public class Game
    {
        private Board Board;
        private Card[] EventCards;
        private Card[] CommunityCards;
        private List<Street> AvailableStreets;
        private List<Player> Players;
        private int Zahl;

        public Game() {
            Init();        
        }

        private void Init()
        {
            EventCards = new Card[16];
            CommunityCards = new Card[16];
            Math.Round(0.2);
        }


        public void ComputeNextRound()
        {

        }

        public void StartNewGame()
        {
        }

        public bool IsGameFinished()
        {
            return false;
        }
    }
}

File: Game.py
import Board
import Card
import Player

class Game:
    def __init__(self, numberPlayers):
        self.Board = Board.Board()
        self.NumberPlayers = numberPlayers
        self.Players=[]
        self.Round=0
        
    def StartNewGame(self):
        return
    
    def IsGameFinished(self):
        return
    
    def GetStreetsOfWinner(self):
        return